### ISSUE TITLE
Start a community Discord server for Talemate

### ISSUE THREAD

**Issue by RavenThunderclaw** [vegu-ai/talemate#145]
The software is extremely obscure right now and a discord group can help a ton.

This will be extremely helpful for people like me who are using ST to transition to Talemate.

This will also bring exposure and we can have prebuilt scenes. Currently I'm not able to find any source for one.

**Comment by Selbstverlust** [comment]
This is probably the project I have been most interested in recent times, I agree with the creation of a Discord server, would help out a lot in building a community and tracking development for people who aren't used to git.

**Comment by Vandread89** [comment]
yes please. a community around this awsome project would be sweet. Just made a git account to say that.

**Comment by vegu-ai-tools** [comment]
https://discord.gg/8bGNRmFxMj
